src/acrcssc/azext_acrcssc/commands.py

```diff
@@ -0,0 +1,20 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+# pylint: disable=line-too-long
+from azext_acrcssc._client_factory import cf_acr
+
+
+def load_command_table(self, _):
+    # required to mark the command as 'preview', can be expanded if additional commands are added
+    self.command_group("acr supply-chain", client_factory=cf_acr, is_preview=True)
+
+    with self.command_group("acr supply-chain workflow", client_factory=cf_acr, is_preview=True) as g:
+        g.custom_command("create", "create_acrcssc")
+        g.custom_command("update", "update_acrcssc")
+        g.custom_command("delete", "delete_acrcssc")
```

Comment:
Do we need to consider adding `confirmation=True` for the delete command?

Reply:
we are doing a direct `user_confirmation()` on the command, did not find any documentation when adding it, but what is the preferred method?
https://github.com/AzureCR/azure-cli-extensions/blob/f48d94961ebad8403ebfe10687ee0d9187d6591e/src/acrcssc/azext_acrcssc/cssc.py#L114C5-L114C22

Reply:
If you want to customize the confirmation information, then using `user_confirmation()` is fine